draft-packages/button/KaizenDraft/Button/styles.scss

```diff
@@ -28,6 +28,14 @@ $caButton-verticalPaddingForm: calc(
   transition: none; // override Murmur global styles :(
 }
 
+@mixin working-state {
+  pointer-events: none;
```

Comment:
Given you have this, perhaps you can actually get rid of the `onClick` override in `GenericButton`?